api/orm/models/employee.py

```diff
@@ -0,0 +1,19 @@
+from sqlalchemy import Column, String, Integer
```

Comment:
Can we please rather have a models.py with the employee model in it (see https://www.python.org/dev/peps/pep-0020/ - "Flat is better than nested".

Reply:
Would re-structure as per EPR #19